auth/session.go

```diff
@@ -30,10 +35,16 @@ func (icentral *ImqsCentral) validateUserIsEnabledForNewLogin(sessionKey string,
 	r.token = token
 	auditUserLogAction(icentral, r, token.UserId, token.Identity, "User Profile: "+token.Identity, authaus.AuditActionAuthentication)
 	permList, err := authaus.PermitResolveToList(token.Permit.Roles, icentral.Central.GetRoleGroupDB())
-	if err != nil {
+	if err != nil && permList == nil {
+		icentral.Central.Log.Errorf("validateUserIsEnabledForNewLogin: Failed to resolve permit, returning (%v)", err)
 		return nil, fmt.Errorf("Failed to resolve permit: %w", err)
-	} else if !permList.Has(PermEnabled) {
-		return permList, ErrUserDisabled
+	} else {
+		if err != nil {
+			icentral.Central.Log.Errorf("validateUserIsEnabledForNewLogin: Failed to resolve permit, continuing (%v)", err)
```

Comment:
```suggestion
			icentral.Central.Log.Warnf("validateUserIsEnabledForNewLogin: Failed to resolve permits: %v", err)
```

Reply:
Will add error logging to the calling functions.

Reply:
Due the the semantics of the function itself, the error will be hidden by using another error to indicate the answer to the question. Dubious practise in my opinion - but did not want to rewrite this unnecessarily. Had to add quite a bit of extra code to handle the different cases properly.